robottelo/ui/locators/base.py

```diff
@@ -1754,22 +1758,23 @@
         "//button[contains(@ng-show, 'list') and "
         "contains(@ng-click, 'removeRepositories')]"),
     "contentviews.repo_search": (
-        By.XPATH, "//input[@ng-model='repositorySearch']"),
+        By.XPATH, "//input[@ng-model='table.searchTerm']"),
```

Comment:
I have not checked yet all the pages. but looks like all searchbars now has same ng-model, `table.searchTerm` myabe it would be better to create new entry in common locators?